content/telegraf/v1.21/about_the_project/release-notes-changelog.md

```diff
@@ -10,6 +10,51 @@ menu:
     weight: 10
     parent: About the project
 ---
+## v1.21.4 [2022-2-16]
+
+- Update all `go.opentelemetry.io` from 0.24.0 to 0.27.0.
```

Comment:
Can you add "Update to Go 1.17.7 to addressed [three security issues](https://groups.google.com/g/golang-announce/c/SUsQn0aSgPQ/m/gx45t8JEAgAJ?pli=1) in the library" to the top